content/en/docs/setup/install/operator/index.md

```diff
@@ -239,76 +239,76 @@ $ kubectl get pods --namespace istio-system \
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=verify_operator_cr >}}
 $ kubectl get iop --all-namespaces
 NAMESPACE      NAME                        REVISION   STATUS    AGE
 istio-system   example-istiocontrolplane              HEALTHY   11m
 {{< /text >}}
 
 Download and extract the `istioctl` corresponding to the version of Istio you wish to upgrade to.
 Then, run the following command to install the new target revision of the Istio control plane based on the in-cluster
-`IstioOperator` CR (here, we assume the target revision is 1.8.1):
+`IstioOperator` CR (here, we assume the target revision is {{< istio_full_version_revision >}}):
 
-{{< text bash >}}
-$ istio-1.8.1/bin/istioctl operator init --revision 1-8-1
+{{< text syntax=bash snip_id=canary_upgrade_init >}}
+$ istio-{{< istio_full_version >}}/bin/istioctl operator init --revision {{< istio_full_version_revision >}}
 {{< /text >}}
 
 {{< tip >}}
 You can alternatively use Helm to deploy another operator with a different revision setting:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=canary_upgrade_helm_install >}}
 $ helm install istio-operator manifests/charts/istio-operator \
   --set watchedNamespaces=istio-system \
   -n istio-operator \
-  --set revision=1-9-0
+  --set revision={{< istio_previous_version_revision >}}-1
 {{< /text >}}
 
 Note that you need to [download the Istio release](/docs/setup/getting-started/#download)
 to run the above command.
 {{< /tip >}}
 
-Make a copy of the `example-istiocontrolplane` CR and save it in a file named `example-istiocontrolplane-1-8-1.yaml`.
-Change the name to `example-istiocontrolplane-1-8-1` and add `revision: 1-8-1` to the CR.
+Make a copy of the `example-istiocontrolplane` CR and save it in a file named `example-istiocontrolplane-{{< istio_full_version_revision >}}.yaml`.
+Change the name to `example-istiocontrolplane-{{< istio_full_version_revision >}}` and add `revision: {{< istio_full_version_revision >}}` to the CR.
 Your updated `IstioOperator` CR should look something like this:
 
-{{< text bash >}}
-$ cat example-istiocontrolplane-1-8-1.yaml
+{{< text syntax=bash snip_id=cat_operator_yaml >}}
+$ cat example-istiocontrolplane-{{< istio_full_version_revision >}}.yaml
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 metadata:
   namespace: istio-system
-  name: example-istiocontrolplane-1-8-1
+  name: example-istiocontrolplane-{{< istio_full_version_revision >}}
 spec:
-  revision: 1-8-1
+  revision: {{< istio_full_version_revision >}}
   profile: demo
 {{< /text >}}
 
 Apply the updated `IstioOperator` CR to the cluster. After that, you will have two control plane deployments and services running side-by-side:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=get_pods_istio_system >}}
 $ kubectl get pod -n istio-system -l app=istiod
 NAME                            READY   STATUS    RESTARTS   AGE
```

Comment:
Nit: the format is slightly off in the preview.
```suggestion
NAME                             READY   STATUS    RESTARTS   AGE
```

Reply:
addressed in 4f26c6b74399680e5a7bd1b49f57fd72561b7546